keras_cv/layers/video_swin_layers.py

```diff
@@ -0,0 +1,980 @@
+# Copyright 2024 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from keras import layers
```

Comment:
Let's remove this import and use `keras.layers` instead. This is a good pattern to follow until Keras 2 support is dropped. We can then get rid of the `from keras_cv.backend import keras` and do keras imports directly.